README.md

```diff
@@ -217,8 +239,9 @@ Firstly thanks!  We value your time and will do our best to review the PR as soo
 4. Run the tests by `$ make test`
 5. Setup environment variable `TF_ACC` to enable online acceptance test cases by `$ export TF_ACC=1`
 6. Setup environment variables `IC_USERNAME` and `IC_API_KEY` of your provisioning API to grant online acceptance test cases permission by `$ export IC_USERNAME=<your instaclustr username>` and `$ export IC_API_KEY=<your provisioning API key>`
-7. Run the acceptance tests `$ make testacc`
-8. Create a PR and send it our way :)
+7. Setup environment variable `KMS_ALIAS` and `KMS_ARN` to your AWS KMS key values to run test cases for EBS encryption of nodes by `$ export KMS_ALIAS=<your KMS alias>` and `$ export KMS_ARN=<your KMS ARN>`
```

Comment:
Could KMS_ALIAS not be set by the user? It could be defaulted to any string.

Reply:
Re: this point - if KMS_ALIAS isn't necessary, it could be removed from the code as well and you could instead just set the alias in the .tf file

Reply:
Are KMS alias' mutable? i thought they were specific to the key-gen from AWS and as such must match the ARN. If not then i'll just change it to a default string like `instaclustr-kms-alias` or something like that.

Reply:
The alias used here isn't to identify the key with AWS but for display purposes, such as when you see the list of keys when creating a cluster via the console. So yes changing it to a default value would be good, because it's 1 less environment variable to set.